### ISSUE TITLE
Cannot sort class in .liquid file when passing "functions"

### ISSUE THREAD

**Issue by andershagbard** [tailwindlabs/prettier-plugin-tailwindcss#141]
Config:
```json
{
  "plugins": ["@shopify/prettier-plugin-liquid", "prettier-plugin-tailwindcss"]
}
```

Input:
```liquid
<div class="flex bg-green-500 px-2 gap-10 py-10 pb-14 -mx-10 {{ foo }}"></div>
<div class="flex bg-green-500 px-2 gap-10 py-10 pb-14 -mx-10 {% render 'foo' %}"></div>
<div class="flex bg-green-500 px-2 gap-10 py-10 pb-14 -mx-10 {% render 'foo', bar: true %}"></div>
<div class="flex bg-green-500 px-2 gap-10 py-10 pb-14 -mx-10 {% include 'foo' %}"></div>
<div class="flex bg-green-500 px-2 gap-10 py-10 pb-14 -mx-10 {% include 'foo', bar: true %}"></div>
```

Expected output:
```liquid
<div class="-mx-10 flex gap-10 bg-green-500 px-2 py-10 pb-14  {{ foo }}"></div>
<div class="-mx-10 flex gap-10 bg-green-500 px-2 py-10 pb-14  {% render 'foo' %}"></div>
<div class="-mx-10 flex gap-10 bg-green-500 px-2 py-10 pb-14  {% render 'foo', bar: true %}"></div>
<div class="-mx-10 flex gap-10 bg-green-500 px-2 py-10 pb-14  {% include 'foo' %}"></div>
<div class="-mx-10 flex gap-10 bg-green-500 px-2 py-10 pb-14  {% include 'foo', bar: true %}"></div>
````

Actual output:
```liquid
<div class="-mx-10 flex gap-10 bg-green-500 px-2 py-10 pb-14 {{ foo }}"></div>
<div class="flex bg-green-500 px-2 gap-10 py-10 pb-14 -mx-10 {% render 'foo' %}"></div>
<div class="flex bg-green-500 px-2 gap-10 py-10 pb-14 -mx-10 {% render 'foo', bar: true %}"></div>
<div class="flex bg-green-500 px-2 gap-10 py-10 pb-14 -mx-10 {% include 'foo' %}"></div>
<div class="flex bg-green-500 px-2 gap-10 py-10 pb-14 -mx-10 {% include 'foo', bar: true %}"></div>
```

**Comment by thecrypticace** [comment]
Hey, thanks for reporting this. I've merged a fix for this in #143 and will be in the next release (will be soon).

You can try it out before we do a release by using our insiders build: `npm install prettier-plugin-tailwindcss@insiders`

**Comment by andershagbard** [comment]
Thanks for the fast fixes. Very appreciated!
